apps/docs/content/tokens/semantic/typography.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Typography
+description: Getting started with Workleap Design Tokens
+---
+import tokens from '../../../../../packages/tokens/src/tokens/semantic/light/fonts.tokens.json';
```

Comment:
The datas must be taken from the `'../../../datas/tokens.json` file and not from the token package.

Reply:
/datas/tokens.json does not expose the composite tokens. I can deduce it from the tokens.json but I found that it was more legible and easier to work with already constructed tokens.

```
    "heading": {
        "3xl": {
            "font-family": {
                "$type": "fontFamily",
                "$value": "{font.family.primary}"
            },
            "font-size": {
                "$type": "fontSize",
                "$value": "{font.size.360}"
            },
            "font-weight": {
                "$type": "fontWeight",
                "$value": "{font.weight.680}"
            },
            "line-height": {
                "$type": "lineHeight",
                "$value": "{line-height.1-33}"
            }
        },
        "2xl": {
            "font-family": {
                "$type": "fontFamily",
                "$value": "{font.family.primary}"
            },
            "font-size": {
                "$type": "fontSize",
                "$value": "{font.size.320}"
            },
            "font-weight": {
                "$type": "fontWeight",
                "$value": "{font.weight.580}"
            },
            "line-height": {
                "$type": "lineHeight",
                "$value": "{line-height.1-25}"
            }
        },
```
vs
```
      {
        "name": "hop-body-xs-font-size",
        "value": "0.75rem"
      },
      {
        "name": "hop-body-xs-medium-font-size",
        "value": "0.75rem"
      },
      {
        "name": "hop-body-xs-semibold-font-size",
        "value": "0.75rem"
      },
      {
        "name": "hop-body-xs-bold-font-size",
        "value": "0.75rem"
      },
      {
        "name": "hop-body-xs-underline-font-size",
        "value": "0.75rem"
      },
```

Reply:
I understand your pain. You could take a look at tableSection to see how I had to do it working with tokens.json